### PR TITLE
Upload coverage on master an push CI checks

### LINKED ISSUE #1666: Coverage test passes even if coverage goes down
I've noticed that all recent pull requests pass coveralls test and get the green icon regardless of the coverage. Shouldn't the test show the red cross if coverage goes down? 

Suggested tests for the PR fixing this:
1. Remove 2 tests. Expected result--> Coveralls fails because of coverage down (coveralls report published)
1. Reintroduce 1 test. Expected result--> Coveralls fails because of coverage down in comparison with master (coveralls report updated)
1. Reintroduce last test -> Coveralls passes because coverage doesn't change from master. (coveralls report updated).

### PR DESCRIPTION
**Pull request by patrikgrenfeldt**
This PR fixes the issue with coverage reports missing on PRs (fix #1666)

**How to test**:
1. Open this PR on Github
1. Commit some changes that reduces the amount of code covered
1. Revert those changes

**Expected outcome**:
- [x] The build should fail due to decreased coverage
Take a screenshot and attach or copy/paste the output.

**Review:**
- [x] code approved by CR
- [x] tests executed by Travis CI

This is a patch/fix

### CODE REVIEW

**Comment by northwestwitch** [comment]
Works!! 👍

File: .travis.yml [hunk 1]
```diff
@@ -6,20 +6,20 @@ env: TRAVIS=1
...
 
         - name: "Tests & Coverage"
-          if: NOT type = push
+          if: NOT type = pull_request
           services: mongodb
           install:
               - pip install cython
               - pip install -r requirements-dev.txt -e .
-          script: py.test --cov=scout -rxs tests/
+          script: py.test --cov=scout -rxs tests/commands/
```
Comment (northwestwitch) [inline .travis.yml]: I think that as soon as you revert this and update the changelog we are good to merge!

**Comment by northwestwitch** [comment]
I've already merged your other PR, I meant the revert and changelog for this one!

**Comment by northwestwitch** [comment]
Let's hurry up before @dnil !!

File: .travis.yml (hunk 1 above)
Comment (patrikgrenfeldt) [reply .travis.yml]: reverted

**Pull request merged**